### ISSUE TITLE
[build/include_what_you_use] wrong recognition

### ISSUE THREAD

**Issue by hbhkx** [cpplint/cpplint#167]
In my file, I would write my own `copy` function without including \<algorithm\> (Because I don't need it indeed):

``` C++
void copy(char* src, char* dst, size_t size) {
    // do some copy
}
```

But after using `cpplint.py`, it raises a warning: `Add #include <algorithm> for copy  [build/include_what_you_use] [4]`.

That's absolutely a wrong recognition, for here is a function definition rather than a function call.
How can I avoid this while I need custom copy function but I really don't need \<algorithm\>?

**Comment by tkruse** [comment]
That's also a good question to ask upstream at https://github.com/google/styleguide/issues. I am not sure if you can avoid the warning, other than maybe call your function differently.
